package.json

```diff
@@ -61,18 +61,19 @@
     "gulp-uglify": "^3.0.1",
     "husky": "^1.1.1",
     "lerna": "^3.0.0-rc.0",
-    "lint-staged": "^7.2.0",
```

Comment:
@mwcz Is lint-staged being removed because you're minifying during commit so auto-linting is not necessary?

Reply:
I see the `pretty-quick` tool replaces `lint-staged` on the pre-commit task; safe to assume it's doing the same clean-up just on certain files and not the compiled assets?

Reply:
No change in functionality, just a different tool for the same job.  The tagline of lint-staged is "Make Linting Great Again", need I say more?

Reply:
+1 please don't say more 😛